Makefile

```
@@ -52,7 +51,7 @@ lint-frontend:  ## Run front-end linters
 
 .PHONY: test
 test:  ## Run the tests and check coverage.
-	poetry run pytest -n auto --ds=cms.settings.test --cov=cms --cov-report term-missing
+	poetry run coverage run ./manage.py test --parallel --settings=cms.settings.test
```

```suggestion
	poetry run coverage run ./manage.py test --parallel --settings=cms.settings.test
	poetry run coverage report
```

also we may need something like
```
branch = true
parallel = true
concurrency = multiprocessing, thread
``` 
in the coverage run configuration to better work with parallel runs. https://coverage.readthedocs.io/en/7.6.4/config.html#run-parallel

+ `python run coverage combine` before `poetry run coverage report`

---

https://stackoverflow.com/questions/42721896/coverage-in-parallel-for-django-tests has some notes too

---

Parallel and sequential test runs are now reporting the same coverage numbers :partying_face: